docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -100,6 +100,21 @@ Some constraints are due to differences in behavior between Docker and Singulari
    character (") may cause the job to fail with the json.decoder.JSONDecodeError unless you have
    Singularity 3.10 or greater or Apptainer 1.1 or greater.
 
+************************
+ Apptainer Known Issues
+************************
+
+Starting with Apptainer version 1.1.0 some changes may trigger permission problems inside of
+Determined containers for shells, tensorboards, and experiments. For example, a tensorboard log may
+contain ``ERROR: Could not install packages due to an OSError: [Errno 28] No space left on device``,
+or a shell may fail to function and the shell logs contain the message ``chown(/dev/pts/1, 63200, 5)
+failed: Invalid argument``, or an experiment may fail to launch due to ``FATAL: container creation
+failed: mount /var/tmp->/var/tmp error: while mounting /var/tmp: could not mount /var/tmp: operation
+not supported``. This likely indicates an installation or configuration error for unprivileged
+containers. Review the :ref:Installing Apptainer
```

Comment:
The link is not generated properly.    An external link is apparently of the form:
```
 `Slurm <https://slurm.schedmd.com/documentation.html>`__
```


![image](https://user-images.githubusercontent.com/84593277/213450636-953737b3-3925-4f47-a078-68409cc0f9be.png)

Reply:
Fixed. Thanks.